### PR TITLE
Stocks content fixes

### LINKED ISSUE #2079: Stocks: Form type 0005 and 0006 sending incorrect dump to SDX
### Expected behavior
the submission dump must be sent to SDX for transformation

### Actual behavior
SDX failing to transform as it has received incorrect submission dump from eQ

q_code 189 was missing and 180 has two values associated against it.

### Steps to reproduce the behavior
Launch stocks_0005
answer all the questions 
and submit

### Technical information
{
submission: {
case_id: "0bf22128-0774-42e5-be48-b34807182cc1",
collection: {
exercise_sid: "7504b22c-2421-4caf-b5cf-3080a985cf50",
instrument_id: "0005",
period: "201605"
},
data: {
174: "13123",
175: "12412",
179: "14124",
180: [
"141241",
"434234"
],
184: "1241124",
185: "12414",
190: "23523",
191: "13123",
192: "3123"
},
flushed: false,
metadata: {
ref_period_end_date: "2016-05-31",
ref_period_start_date: "2016-05-01",
ru_ref: "22521433310H",
user_id: "UNKNOWN"
},
origin: "uk.gov.ons.edc.eq",
started_at: "2019-05-15T12:58:30.580454",
submitted_at: "2019-05-15T13:02:38.526489",
survey_id: "017",
tx_id: "7e136779-4650-4020-9a50-887c05244363",
type: "uk.gov.ons.edc.eq:surveyresponse",
version: "0.0.1"
}
}

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
This PR is to fix some content errors with the stocks survey
Fixes #2079

### How to review 
Make sure survey launches and all content problems are resolved

### CODE REVIEW

File: data/en/stocks_0013.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
-    "eq_id": "512",
+    "eq_id": "87935e77-e71d-4435-99ed-520917b03692",
```
Comment (samiwel) [inline data/en/stocks_0013.json]: I wonder if this should be `stocks`?  Just looking at QPSES and LMS schemas and the `eq_id` seems to be the name of the survey for the most case.

**Review by samiwel — COMMENTED** [review]
Changes look okay to me. Just a general comment about the eq_id

**Review by pricea1 — CHANGES_REQUESTED** [review]
Looks good but tests are failing

**Comment by samiwel** [comment]
@rmccar Pushed up a commit to remove all the markup from the titles. It was easier to do that than adding review comments.

**Review by samiwel — APPROVED** [review]

**Comment by rmccar** [comment]
[eq - survey - runner. Stocks: Form type 0013 and 0014 are incorrect..](https://trello.com/c/WAXPDSSR/1119-eq-survey-runner-stocks-form-type-0013-and-0014-are-incorrect)

File: data/en/stocks_0010.json [hunk 2]
```diff
@@ -257,7 +254,7 @@
                                     "id": "answer6061",
                                     "mandatory": true,
                                     "type": "Currency",
-                                    "label": "Value at start of period",
+                                    "label": "VValue of other fuel stocks at start of period",
```
Comment (pricea1) [inline data/en/stocks_0010.json]: typo

File: data/en/stocks_0010.json [hunk 3]
```diff
@@ -320,7 +317,7 @@
                                     "id": "answer6056",
                                     "mandatory": true,
                                     "type": "Currency",
-                                    "label": "Value at end of period",
+                                    "label": "Value of stroes at end of period",
```
Comment (pricea1) [inline data/en/stocks_0010.json]: typo

File: data/en/stocks_0013.json [hunk 4]
```diff
@@ -133,19 +206,19 @@
...
                             "id": "value-stocks-materials",
                             "type": "Question",
                             "questions": [{
-                                "id": "question4731",
+                                "id": "question4884",
                                 "title": "What was the value of stocks held for <em>materials, stores, and fuel</em> (net of progress payments on long-term contracts)?",
                                 "guidance": {
                                     "content": [{
-                                            "title": "Include"
+                                            "description": "Include"
```
Comment (pricea1) [inline data/en/stocks_0013.json]: should this be "title" like the others?

File: data/en/stocks_0013.json [hunk 5]
```diff
@@ -159,7 +232,7 @@
                                             ]
                                         },
                                         {
-                                            "title": "Exclude"
+                                            "description": "Exclude"
```
Comment (pricea1) [inline data/en/stocks_0013.json]: should this be "title" like the others?

File: data/en/stocks_0014.json [hunk 6]
```diff
@@ -133,19 +206,19 @@
...
                             "id": "value-stocks-materials",
                             "type": "Question",
                             "questions": [{
-                                "id": "question4731",
+                                "id": "question4884",
                                 "title": "What was the value of stocks held for <em>materials, stores, and fuel</em> (net of progress payments on long-term contracts)?",
                                 "guidance": {
                                     "content": [{
-                                            "title": "Include"
+                                            "description": "Include"
```
Comment (pricea1) [inline data/en/stocks_0014.json]: should this be "title" like the others?

**Review by pricea1 — CHANGES_REQUESTED** [review]
Just a few minor amends

**Review by pricea1 — APPROVED** [review]
LGTM

**Pull request merged**